site/_variables.yml

```diff
@@ -1,19 +1,44 @@
-# Product names
-vm_platform: "ValidMind Platform UI"
-vm_framework: "ValidMind Developer Framework"
-vm_ui: "platform UI"
-vm_dev: "developer framework"
-vm_risk: "ValidMind AI risk platform"
-
-# Product URLs
+# VARIABLE NAMING CONVENTIONS
+# - Variables for long & short forms should be related (e.g. `developer_framework`, `developer`)
+# - Variable names should be descriptive and consistent (e.g. `user_first_name`, `user_last_name`)
+# - Use prefixes / suffixes for grouping (e.g. `orderDate`, `orderAmount`)
+# - Use snake case (e.g. `developer_framework`)
+# - Avoid abbreviations (e.g. `customer_address` NOT `custAddr`) — `vm` for ValidMind is one exception
+# - Include enough contextual information (e.g. `invoiceTotal` NOT `total`)
+
```

Comment:
Minor: there's a lotta blank lines in this file, I'd maybe not have more than one between sections.

Reply:
Hm, this is again a visual parsing quirk of mine (to make the sections more discrete since there isn't any other way to format them) but if it interferes with how the variables work I can definitely adjust. 🤔